media/css/products/affiliate.scss

```diff
@@ -1,60 +0,0 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Note for reviewer: I removed this file as it seems to be a duplicate? The actual file in use seems to be here: https://github.com/mozilla/bedrock/blob/main/media/css/products/shared/affiliate.scss